src/h5web/dataset-visualizer/vis/matrix/IndexedGrid.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState, forwardRef, Ref } from 'react';
+import { FixedSizeGrid as Grid, GridChildComponentProps } from 'react-window';
+import StickyProvider from './StickyContext';
+import styles from './MatrixVis.module.css';
+import AnchorCell from './AnchorCell';
+import IndexTrack from './IndexTrack';
+import { GridSettings } from './utils';
+
+interface Props {
+  children: (props: GridChildComponentProps) => JSX.Element;
+  itemData?: any; // eslint-disable-line @typescript-eslint/no-explicit-any
+  gridSettings: GridSettings;
+  width: number;
+  height: number;
+}
+
+// IndexedGrid based on https://codesandbox.io/s/0mk3qwpl4l
+// Wrapper around FixedSizeGrid to add indices row and column (sticky or not).
+function IndexedGrid(props: Props): JSX.Element {
+  const {
+    children: NormalCellFunc,
+    width,
+    height,
+    gridSettings,
+    itemData,
```

Comment:
Here, I use `itemData` to pass the accessor function from the `Matrix` component down to the `ValueCell` component. `itemData` is a  `react-window` props described as a light-weight alternative to React's `Context`. But I wonder if using a `Context` would be easier here.

Reply:
Yeah that means they just pass down the prop themselves throught their internal components. Putting it in the context would be more consistent and less obscure.

Reply:
Ignore this comment, don't put it into a context, there's no need. You don't even need to pass `itemData` to react-window's `Grid`, since you now declare the cell renderer is no longer a separate component. You can use the prop directly in the renderer's code:

```
<Grid ...>
  {
    ...
    return <NormalCellFunc data="{itemData} ...>...</NormalCellFunc>
  }
</Grid>
```